config/install/apigee_edge.error_page.yml

```diff
@@ -3,3 +3,4 @@ error_page_title: 'Connection error'
 error_page_content:
  value: 'Cannot reach management server. An administrator should check the configuration of this site.'
  format: 'plain_text'
+error_page_debug_messages: true
```

Comment:
An update hook is missing from this PR that initiates this configuration on existing sites.